tests/python/gaia-ui-tests/gaiatest/tests/functional/ftu/test_ftu_skip_tour.py

```diff
@@ -101,9 +110,17 @@ def test_ftu_skip_tour(self):
         self.ftu.tap_next_to_firefox_accounts_section()
         self.ftu.tap_next_to_welcome_browser_section()
 
-        # Tap the statistics box and check that it sets a setting
-        self.ftu.tap_statistics_checkbox()
-        self.assertTrue(self.data_layer.get_setting('debug.performance_data.shared'))
+        # Verify the statistics checkbox reflects the initial internal state
+        Wait(self.marionette).until(
+            lambda m: self.ftu.is_statistics_enabled == share_data_default,
+            message="Send Data checkbox should match initial internal state")
+
+        # Tap the statistics checkbox and verify that it toggled internal state
+        self.ftu.toggle_statistics()
+        self.assertEqual(self.data_layer.get_setting(
```

Comment:
Like said in bug 1134445, this test showed that the default value has changed in the product, which is an actual bug. To answer on comment 8, I would leave the hardcoded value here.